bundles/org.openhab.binding.homeconnect/README.md

```diff
@@ -110,6 +110,9 @@ After the bridge has been added and authorized, devices are discovered automatic
     for example: `http://192.168.178.34:8080/homeconnect` or `https://myhome.domain.com/homeconnect`
     - _One Time Token Mode_: keep unchecked
     - _Proof Key for Code Exchange_: keep unchecked
+      
+       > **NOTE**: You might get an error (`403 - Forbidden`) in case you entered a bare IP of your local openhab instance. In that case, use your instance's hostname instead (e.g. for openhabian `http://openhabian:8080/homeconnect`) and try again.
```

Comment:
```suggestion
       > **NOTE**: You might get an error (`403 - Forbidden`) in case you entered a bare IP of your local openHAB instance. In that case, use your instance's hostname instead (e.g. for openhabian `http://openhabian:8080/homeconnect`) and try again.
```